tests/components/vera/test_init.py

```diff
@@ -0,0 +1,434 @@
+"""Vera tests."""
+
+import requests_mock
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_registry import async_get_registry
+
+from .common import (
+    DEVICE_ALARM_SENSOR_ID,
+    DEVICE_CURTAIN_ID,
+    DEVICE_DIMMER_ID,
+    DEVICE_DOOR_SENSOR_ID,
+    DEVICE_HUMIDITY_SENSOR_ID,
+    DEVICE_IGNORE,
+    DEVICE_LIGHT_ID,
+    DEVICE_LIGHT_SENSOR_ID,
+    DEVICE_LOCK_ID,
+    DEVICE_MOTION_SENSOR_ID,
+    DEVICE_POWER_METER_SENSOR_ID,
+    DEVICE_SCENE_CONTROLLER_ID,
+    DEVICE_SWITCH2_ID,
+    DEVICE_SWITCH_ID,
+    DEVICE_TEMP_SENSOR_ID,
+    DEVICE_THERMOSTAT_ID,
+    DEVICE_UV_SENSOR_ID,
+    RESPONSE_LU_SDATA_EMPTY,
+    RESPONSE_SDATA,
+    RESPONSE_STATUS,
+    assert_state,
+    async_call_service,
+    async_configure_component,
+    get_device,
+    get_entity_id,
+    update_device,
+)
+
+
+async def test_full(hass: HomeAssistant) -> None:
```

Comment:
Please break this test up into different tests and put the tests for different platforms in different modules.